src/clusterfuzz/_internal/build_management/source_mapper.py

```diff
@@ -183,8 +187,29 @@ def get_vcs_viewer_for_url(url):
   return None
 
 
+def should_linkify_java_stack_frames():
+  return local_config.Config(local_config.PROJECT_PATH).get('linkify_java')
```

Comment:
This may be causing some performance regressions if this is reading in the config from a file for every single line.